src/calendars/slovakia.ts

```diff
@@ -181,41 +181,35 @@ const dates = async (config: RomcalConfig): Promise<Array<RomcalLiturgicalDayInp
       liturgicalColors: LiturgicalColors.WHITE,
     },
     {
-      key: 'dedicationOfAParticularChurch',
+      key: 'dedication_of_consecrated_churches',
       rank: Ranks.SOLEMNITY,
       date: dayjs.utc(`${year}-10-26`),
       liturgicalColors: LiturgicalColors.WHITE,
     },
     {
-      key: 'allSouls',
+      key: 'all_souls',
       rank: Ranks.MEMORIAL,
       date: dayjs.utc(`${year}-11-2`),
       liturgicalColors: [LiturgicalColors.PURPLE, LiturgicalColors.BLACK],
     },
     {
-      key: 'saintEmeric',
+      key: 'emeric_of_hungary',
       rank: Ranks.OPT_MEMORIAL,
       date: dayjs.utc(`${year}-11-5`),
     },
-    {
-      // This key is dropped only because of adding Optional Memorial of St Barbara to the same day
-      key: 'saintJohnDamascenePriestAndDoctor',
-      date: dayjs.utc(`${year}-12-4`),
-      drop: true,
-    },
```

Comment:
@emagnier, as this celebration is actually not dropped (it defined in the following lines of `slovakia.ts`), the test of dropping a celebration in `src/lib/calendar-builder.test.ts` (L330-346) should fail, but it does not fail for some reason. The test (if needed) should be rewritten using a celebration from from `test.ts` calendar. Actually, currently we don’t a calendar that would drop a celebration. Originally, @pejulian implemented this functionality because romcal (back then) did not support multiple celebrations on a single day. Therefore we might consider removing `drop` functionality altogether.

Reply:
I'm up to remove the drop feature, as it's a bit of a hack, and indeed no more necessary since the support of multiple celebrations on a single day.

Reply:
Actually, as part of dealing with some final issues in this PR, I have made use of `drop`: see [`slovakia.ts` (L25)](https://github.com/romcal/romcal/pull/213/files#diff-9ccf041d33cc709818b5a0b5b69a3b36R25). I dropped the celebration of `cyril_the_philosopher_monk_and_methodius_of_thessaloniki_bishop` which should not be in the calendar (until we implement #110). Also `cyril_the_philosopher_monk_and_methodius_of_thessaloniki_bishop_secondary_patrons_of_europe` should not be celebrated in its 'default' form neither on 14 Feb nor 5 July. On 5 July, we celebrate `cyril_the_philosopher_monk_and_methodius_of_thessaloniki_bishop_slavic_missionaries` which _might be considered the same celebration, however we don't add the patronage title (`secondary patrons of Europe`), which might cause problems when outputing calendars other than `slovakia` in Slovak language (this issue might be solved by implementing #24).